db/model/Gdoc/gdocToArchie.ts

```diff
@@ -111,8 +111,8 @@ function parseParagraph(
 
         let span: Span = { spanType: "span-simple-text", text: content }
 
-        // step through optional text styles to check for an associated URL
-        if (!textRun.textStyle) return span
+        // If there's no URL, or other styling, or the content is just a newline, return without wrapping in anything else
+        if (!textRun.textStyle || content === "\n") return span
```

Comment:
This isn't necessary for the fix, but it was leading to empty `<b>` tags and such, which seemed like HTML pollution for no gain.